wgpu/backends/wgpu_native/_api.py

```diff
@@ -2276,15 +2295,35 @@ def draw_indexed(
 
     def draw_indexed_indirect(self, indirect_buffer, indirect_offset):
         buffer_id = indirect_buffer._internal
-        # H: void f(WGPURenderPassEncoder renderPassEncoder, WGPUBuffer indirectBuffer, uint64_t indirectOffset)
-        libf.wgpuRenderPassEncoderDrawIndexedIndirect(
-            self._internal, buffer_id, int(indirect_offset)
-        )
+        function = self.__pick_render_function(
+            # H: void f(WGPURenderPassEncoder renderPassEncoder, WGPUBuffer indirectBuffer, uint64_t indirectOffset)
+            render_pass=libf.wgpuRenderPassEncoderDrawIndexedIndirect,
+            # H: void f(WGPURenderBundleEncoder renderBundleEncoder, WGPUBuffer indirectBuffer, uint64_t indirectOffset)
+            render_bundle=libf.wgpuRenderBundleEncoderDrawIndexedIndirect,
+        )
+        function(self._internal, buffer_id, int(indirect_offset))
+
+    def __pick_render_function(self, render_pass, render_bundle):
+        if isinstance(self, GPURenderPassEncoder):
+            return render_pass
+        elif isinstance(self, GPURenderBundleEncoder):
+            return render_bundle
+        else:
+            raise NotImplementedError("Unknown render encoder type")
 
 
 class GPUCommandEncoder(
     classes.GPUCommandEncoder, GPUCommandsMixin, GPUDebugCommandsMixin, GPUObjectBase
 ):
+
+    # GPUDebugCommandsMixin
+    # H: void f(WGPUCommandEncoder commandEncoder, char const * groupLabel)
+    push_debug_group_function = libf.wgpuCommandEncoderPushDebugGroup
+    # H: void f(WGPUCommandEncoder commandEncoder)
+    pop_debug_group_function = libf.wgpuCommandEncoderPopDebugGroup
+    # H: void f(WGPUCommandEncoder commandEncoder, char const * markerLabel)
+    insert_debug_marker_function = libf.wgpuCommandEncoderInsertDebugMarker
```

Comment:
I'd prefer if these were prefixed with an underscore, otherwise they look like public API.

Reply:
Moot comment once we move to `self.__pick_render_function()`.